budcamp/Homework3/Homework3/task1/History.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal abstract class History
+    {
+        public string DeadRussians { get; set; }
+        public virtual void PolishLithuanianCommonwealth() 
```

Comment:
There is a convention that method names should be verbs or contain verbs. Please change method name to describe the action it's doing.